src/frontend/src/common/types/bioinformatics.d.ts

```diff
@@ -51,6 +51,13 @@ interface Tree {
   id: number;
 }
 
+interface TemplateArgs {
+  filterPangoLineages?: Array<string>;
+  filterStartDate?: string;
+  filterEndDate?: string;
+  locationId?: string;
```

Comment:
Pretty sure this should be a number. Here's what I'm seeing locally for the API response: 
<img width="294" alt="image" src="https://user-images.githubusercontent.com/89553795/200374238-1b77a885-cfd2-474f-9028-a664a5006fbc.png">

Reply:
Although maybe this will cause some fun annoyances later because you'll need to use that number to do a lookup on an object, so it will be implicitly cast to a string because JavaScript, and perhaps TypeScript will not be happy about that?

Reply:
I thought typescript was mad about this, but apparently not.  It must have been something else.  Changing back to `number` seems to work fine 👍